mlflow/entities/trace.py

```diff
@@ -23,6 +23,9 @@ class Trace(_MlflowObject):
     info: TraceInfo
     data: TraceData
 
+    def __repr__(self) -> str:
+        return f"Trace(request_id={self.info.request_id})"
```

Comment:
i'm condensing the `__repr__` as well because this info is also sent to the notebook. this may affect normal jupyter notebook users, but the previous `repr` wasn't particularly well-formatted either. if users want to inspect the trace further, it's more advisable to do so in mlflow UI or by inspecting `trace.info` or `trace.data` directly.

Reply:
Agreed